test/e2e/catalog_e2e_test.go

```diff
@@ -272,6 +269,15 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 		})
 		Expect(err).ShouldNot(HaveOccurred())
 
+		Eventually(func() types.UID {
+			// Get updated configmap
+			updatedConfigMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
+			Expect(err).ShouldNot(HaveOccurred())
+			return updatedConfigMap.ObjectMeta.OwnerReferences[0].UID
+		}).Should(Equal(fetchedUpdatedCatalog.ObjectMeta.UID))
+		updatedConfigMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
```

Comment:
Alternatively, we could defined the updatedConfigMap upfront and avoid an additional `get` call:
```suggestion
                var updatedConfigMap corev1.ConfigMap
		Eventually(func() types.UID {
		       var err error
			// Get updated configmap
			updatedConfigMap, err = c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
			Expect(err).ShouldNot(HaveOccurred())
			return updatedConfigMap.ObjectMeta.OwnerReferences[0].UID
		}).Should(Equal(fetchedUpdatedCatalog.ObjectMeta.UID))
```

Reply:
Thanks!  It's better.  I'll take your suggested changes.